assets/wizards/payment/index.js

```diff
@@ -1,101 +0,0 @@
-import '../../shared/js/public-path';
-
-/**
- * Payment
- */
-
-/**
- * WordPress dependencies.
- */
-import { Component, render, Fragment, createElement } from '@wordpress/element';
-import { __ } from '@wordpress/i18n';
-
-/**
- * Material UI dependencies.
- */
-import HeaderIcon from '@material-ui/icons/Dns';
-
-/**
- * Internal dependencies.
- */
-import { withWizard } from '../../components/src';
-import Router from '../../components/src/proxied-imports/router';
-import { PaymentMethod } from './views';
-
-/**
- * External dependencies.
- */
-import { loadStripe } from '@stripe/stripe-js';
```

Comment:
`@stripe/stripe-js` should be removed from project's npm dependencies

Reply:
Good catch - removed in https://github.com/Automattic/newspack-plugin/pull/684/commits/444fe2f5b694a092f7a1950d9436f6ae5e697b1e